src/redrose-launcher/main.sh

```
@@ -0,0 +1,80 @@
+mkdir -p ~/.cache
+
+echo -e "${GREEN}Downloading the manifest... ${NC}"
+res=$(curl -s -L -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2026-03-10" https://api.github.com/repos/redroselinux/redroselinux/releases)
+versions=$(echo "$res" | jq ".[].name")
+index=0
+versions_table=()
+# jq .[].assets.[].browser_download_url
+while IFS= read -r line; do
+    versions_table+=("$line")
+    ((index=index+1))
+    trim=$(echo "$line" | tr -d '"\\')
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)